upgrading_appliances/index.md

```diff
@@ -0,0 +1,44 @@
+---
+---
+
+# Upgrading Appliances
+In this example, we will update from jansa to kasparov.
+
+1. Ensure that the appliance is up to date on the current branch
+
+        # dnf update
+
+2. Install the new manageiq-release RPM for the version that you want to upgrade to
```

Comment:
I think we should reference the appendix from here.

Reply:
Done.